### PR TITLE
CSSType. Experimental & non-standard pseudos support  [generated]

### PR DESCRIPTION
**Pull request by aerialist7**

### CODE REVIEW

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-csstype/src/jsMain/generated/csstype/NonStandardRuleBuilder.kt [hunk 1]
```diff
@@ -0,0 +1,165 @@
+// Automatically generated - do not modify!
+
+@file:Suppress(
+    "DECLARATION_CANT_BE_INLINED",
+)
+
+package csstype
+
+interface NonStandardRuleBuilder<T : Any> : RuleBuilder<T> {
```
[the remark this answers is not in the document]
Comment (aerialist7) [reply kotlin-csstype/src/jsMain/generated/csstype/NonStandardRuleBuilder.kt]: Done

File: kotlin-csstype/src/jsMain/generated/csstype/ExperimentalRuleBuilder.kt [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+@file:Suppress(
+    "DECLARATION_CANT_BE_INLINED",
+)
+
+package csstype
+
+import web.cssom.Selector
+
+interface ExperimentalRuleBuilder<T : Any> : RuleBuilder<T> {
```
[the remark this answers is not in the document]
Comment (aerialist7) [reply kotlin-csstype/src/jsMain/generated/csstype/ExperimentalRuleBuilder.kt]: Done

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-csstype/src/jsMain/generated/csstype/ExperimentalRuleBuilder.kt [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        block: T.() -> Unit,
+    ) {
+        "::highlight()($selector)"(block)
+    }
+
+    inline fun highlight(
+        selector: String,
+        block: T.() -> Unit,
+    ) {
+        "::highlight()($selector)"(block)
```
[the remark this answers is not in the document]
Comment (aerialist7) [reply kotlin-csstype/src/jsMain/generated/csstype/ExperimentalRuleBuilder.kt]: Done

File: kotlin-csstype/src/jsMain/generated/csstype/NonStandardRuleBuilder.kt [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+    inline fun mozColorSwatchNonStandard(
```
[the remark this answers is not in the document]
Comment (aerialist7) [reply kotlin-csstype/src/jsMain/generated/csstype/NonStandardRuleBuilder.kt]: Done

**Comment by aerialist7** [comment]
@Leonya, could you please make a release?

**Pull request merged**